docs/content/development/extensions-core/lookups-cached-global.md

```diff
@@ -34,9 +34,22 @@ Globally cached lookups can be specified as part of the [cluster wide config for
          "format": "csv",
          "columns": [
            "key",
-           "value"
+           "val1",
+           "val2"
          ]
        },
+       "maps":[
+         {
+           "mapName":"default",
+           "keyColumn":"key",
+           "valueColumn":"val1"
```

Comment:
When reading through the URIExtractionNamespace changes I now wonder if having the `"maps"` with their keyColumn / valueColumn out here is causing the dizziness and weirdness with simpleJson… because it has no keyColumn!

I wonder if it makes more sense to move `"maps"` into the namespaceParseSpec. That way the parser is in charge of what the map names and k/vs it returns are, and that should remove some of the weirdness in URIExtractionNamespace.